client/my-sites/domains/domain-management/transfer/transfer-to-other-site/transfer-domain-to-other-site.tsx

```diff
@@ -50,10 +50,12 @@ export class TransferDomainToOtherSite extends Component< TransferDomainToOtherS
 	isSiteEligible = ( site: TransferDomainToOtherSiteProps[ 'selectedSite' ] ): boolean => {
 		// check if it's an Atomic site from the site options
 		const isAtomic = site?.options?.is_automated_transfer ?? false;
+		const isWPCOMStaging = site?.is_wpcom_staging_site ?? false;
```

Comment:
It would be good to stick with the naming convention we have in other places e.g., `isWpcomStagingSite`.